src/coq_rules.ml

```diff
@@ -177,20 +177,25 @@ let setup_rules ~sctx ~dir ~dir_contents (s : Dune_file.Coq.t) =
 (* This is here for compatibility with Coq < 8.11, which expects the
    plugin's `.cmxs` files to be in the folder containing the `.vo`
    files *)
-let coq_cmx_install_rules ~dst_dir libs =
+let coq_cmx_install_rules ~package ~dst_dir libs =
```

Comment:
I think it will be a bit cleaner if you compute `ml_libs` in this function. It's the only function that needs this value, so it seems like it should generate it.

Reply:
Indeed, thanks.